pkg/plugins/plugins.go

```diff
@@ -22,6 +29,113 @@ type PluginKey struct {
 	IsInternal     bool
 }
 
+// Implementaion stores implementation type (e.g. instance-storage) and its name (e.g. s3, mongo)
+type Implementaion struct {
```

Comment:
Not a huge deal but this is misspelled, should be `Implementation` (missing the t)

Reply:
🤦